README.md

```diff
@@ -56,8 +56,8 @@
 
```

Comment:
![image](https://github.com/yash244466666/Blog_app/assets/93039359/c4e556ae-a018-4bde-89c8-31b60a2474cd)

- [ ] It would be nice if you fix the Live Demo, Run tests, Deployment, and FAQ links in the Table of contents to make your readme descriptive and professional :+1: If you don't have anything regarding these links then remove these links to make your readme clean and professional :+1: :blush: